carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/core/factory/AbstractFactory.java

```diff
@@ -69,6 +74,22 @@ protected boolean isCapabilitiesEmpty(Capabilities capabilities) {
         return capabilities == null || MapUtils.isEmpty(capabilities.asMap());
     }
 
+    protected void setIdleTimeout(DesiredCapabilities capabilities) {
+        //return if ESG is not used
```

Comment:
we should no verify selenium url value and exclude something... if capability provided we must transfer it into any hub